src/Source/Source.js

```diff
@@ -1,4 +1,26 @@
 import Extent from 'Core/Geographic/Extent';
+import GeoJsonParser from 'Parser/GeoJsonParser';
+import KMLParser from 'Parser/KMLParser';
+import GpxParser from 'Parser/GpxParser';
+import VectorTileParser from 'Parser/VectorTileParser';
+import Fetcher from 'Provider/Fetcher';
+
+export const supportedFetchers = new Map([
```

Comment:
Note: in the future, we should have these two lists in a separate file, where we can put the format itowns supports by default.